docker/entrypoints/uwsgi.sh

```diff
@@ -10,6 +10,17 @@ echo "Waiting for db to be ready..."
 sleep 3
 # makemigrations is needed only for the durin package.
 # The customization of the parameters is not applied until the migration is done
+
+# Installing dependencies for zippy_scan file analyzer
+git clone https://github.com/thinkst/zippy
+cd zippy
+python3 setup.py build 
+python3 setup.py sdist 
+pip3 install dist/*.tar.gz
+cd ..
```

Comment:
nope, this should be added in the Dockerfile, otherwise every time you run the application it would install it again

Reply:
noted!
I was thinking that it would be much easier to maintain if we had all the requirements at one place, so how about i put the script to install zippy inside `requirements/project-requirements/` and add requirements.txt in the same directory as the script as well. This would just require us to make a few more updates inside the docker file and we could have all such kind of dependencies bundled together at one place.